### PR TITLE
Example update and datagen

### PR DESCRIPTION
**Pull request by riteshkarval**

### CODE REVIEW

File: insurance/README.md [hunk 1]
```diff
@@ -47,15 +46,6 @@
   - Datasets: 'insurance-training-data' with version v2. The base dataset is sourced from AWS
   - Model: 'insurance-model' with version v2
 
-
-### Inference
```
Comment (riteshkarval) [inline insurance/README.md:51]: Inference is not required for monitoring demo

File: insurance/data_generation.ipynb [hunk 2]
```diff
@@ -570,42 +558,19 @@
     "            pred_data_name = str(i) + \"_predict_data\"\n",
     "            yield {\"name\": pred_data_name, \"df\": predict_df}\n",
     "    \n",
-    "\n",
-    "    def generate_all_drift(self): ## All features will have drift\n",
```
Comment (riteshkarval) [inline insurance/data_generation.ipynb:574]: This function is no longer in use

File: insurance/README.md [hunk 3]
```diff
@@ -26,8 +26,7 @@
    - Database Name : **
 
 ## Launch IDE
-1. Create an IDE (jupterlab) 
-   - Specify Code as 'insurance' (created above)
+1. Create an IDE (jupterlab)
```
Comment (songole) [inline insurance/README.md]: typo Jupyter?

File: insurance/README.md (hunk 1 above)
Comment (songole) [reply insurance/README.md:51]: We are planning to remove insurance in sklearn branch. We should keep it here

File: insurance/README.md [hunk 4]
```diff
@@ -74,19 +64,22 @@
...
-3. charges as prediction output - continuous
-4. unique_id as RowID - continuous
-5. Timestamp as timestamp - continuous
-6. Change value type: Age and bmi to continuous
-7. Select all Input features and unselect charges, unique_id, and timestamp.
-8. Click Next and save.
+  - charges as prediction output.
+  - unique_id as RowID
+  - Timestamp as timestamp
+3. Select all or interested Input features and unselect charges, unique_id, and timestamp.
```
Comment (songole) [inline insurance/README.md]: is there a reason to unselect. we can ignore right?

File: insurance/README.md [hunk 5]
```diff
@@ -118,12 +111,17 @@
 - **Prediction Column Name**: charges
 
 ### Alerts
-Add Feature Alerts 
+Add Feature Drift Alerts 
  - The datageneration script will be generating drift on the following features - age, sex, bmi, region. 
  - Suggest to configure a separate alert for each individual feature. 
- - Use a threshold between 0 to 1. generally advised 0.02 to 0.03 inclusive.
+ - Use a threshold between 0 to 1. generally advised 0.5 to 0.6 inclusive.
```
Comment (songole) [inline insurance/README.md]: If I pick all continuous & use KS, what is the default value?
If I pick all categorical & use Chi Squared, what is the default value?
Our recommendation about be in the ball part right?

File: insurance/README.md [hunk 6]
```diff
@@ -118,12 +111,17 @@
...
  - It fires an alert when calculated drift goes under the configured threshold
 
+Add Performance Decay Alerts
+  - Create an alert and choose Perormance Decay from dropdown.
+  - Seelct absolute and choose mse from down.
```
Comment (songole) [inline insurance/README.md]: Can we pick percentage? Absolute is such a huge number.

**Pull request merged**